.github/workflows/source-tests.yml

```diff
@@ -30,6 +30,56 @@ jobs:
           THIS_VERSION: ${{ steps.pr_info.outputs.version }}
           OBJETIVO: ${{ steps.pr_info.outputs.objetivo }}
         run: scripts/check-version
+      - name: Comprueba que el PR no viene de la rama main
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const {all_good, sorry } = require("./lib/mensajes.js")
+            const rama = "${{ steps.pr_info.outputs.rama }}"
+            const prNumber = ${{ github.event.number }}
+            
+            console.log( "» Comprobando rama del PR: " + rama )
+            
+            if ( rama === "main" || rama === "master" ) {
+              // Comentar en el PR explicando el problema
+              await github.rest.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: prNumber,
+                body: "🍋💥❌ **Pull Request cerrado automáticamente**\\n\\n" +
```

Comment:
My point is that you could call `sorry` from here instead of inserting its code...